http4s-client/src/main/scala/com/qvantel/jsonapi/client/http4s/JsonApiInstances.scala

```diff
@@ -0,0 +1,79 @@
+package com.qvantel.jsonapi.client.http4s
+
+import scala.language.higherKinds
+
+import scala.util.{Failure, Success, Try}
+import cats.Applicative
+import cats.data.EitherT
+import cats.effect._
+import cats.implicits._
+import org.http4s.{Charset, DecodeFailure, DecodeResult, EntityDecoder, EntityEncoder, MalformedMessageBodyFailure, MediaType, Message}
+import org.http4s.headers.`Content-Type`
+import spray.json._
+import spray.json.JsonParser.ParsingException
+
+import com.qvantel.jsonapi.{JsonApiReader, JsonApiWriter, readCollection, readOne, rawOne, rawCollection}
+
+final case class Include(include: Set[String])
+
+trait JsonApiInstances {
+  val mediaType: MediaType = MediaType.fromKey(("application", "vnd.api+json"))
```

Comment:
Maybe add `application/vnd.api+json` to http4s `MediaType` via a PR?

Reply:
Made a pr. I will merge this pr and will update it to the actual one if/when it gets in and a new version is published.

https://github.com/http4s/http4s/pull/1569